rust/test

```diff
@@ -134,6 +134,18 @@ test_indexer_cli_reports() {
     # Indexer server cli subcommand works with default args
     idxr_server --help 2>/dev/null
 
+    idxr_server start --help 2>&1 |
+        grep -iq "Usage: mina-indexer server start"
+
+    idxr_server config --help 2>&1 |
```

Comment:
When one "configures" a server, does one also start it?

Reply:
Yes

Reply:
Confusing. See above.